debezium-storage/debezium-storage-redis/src/main/java/io/debezium/storage/redis/offset/RedisOffsetBackingStore.java

```diff
@@ -57,8 +70,21 @@ public synchronized void start() {
         this.load();
     }
 
+    public synchronized void start_no_load() {
```

Comment:
Please use Jana naming convention. Also I see it used fro testing only. If it is the case then please either reduce the visibility or mark it with `@VisibleForTesting` annotation.

Reply:
Apologies. Switching from Python to Java, back and forth. This is fixed